tests/test_regions.py

```diff
@@ -47,7 +47,8 @@
         ],
         'segments': [
             Segment(0, 0x400000, 0x804, 0x804),
-            Segment(0xe10, 0x600e10, 0x228, 0x230),
+            Segment(0xe10, 0x600e10, 0x1f0, 0x1f0),
+            Segment(0x1000, 0x601000, 0x38, 0x40),
```

Comment:
does this test happen to be on a binary with RELRO?

Reply:
Yes -- that is why the ground truth changed.  I wasn't sure if we should also modify the test to check the permissions of the segments.